bzt/modules/locustio.py

```diff
@@ -90,7 +90,8 @@ def startup(self):
 
         wrapper = os.path.join(get_full_path(__file__, step_up=2), "resources", "locustio-taurus-wrapper.py")
 
-        self.env.set({"LOCUST_DURATION": dehumanize_time(load.duration)})
+        if load.duration:
```

Comment:
Why have this logic?

Reply:
This is more of an accidental fix that slipped in this PR.

Without it, if Locust is given `iterations`, but no `hold-for`/`ramp-up` — it finishes right after the first request, as it believes that is has a duration limit.

One way to fix that is to pass `LOCUST_DURATION` only when it's a positive number.